dashboard/test/lib/expired_deleted_account_purger_test.rb

```diff
@@ -315,6 +338,12 @@ def setup
       deleted_before: #{15.days.ago}
       max_accounts_to_purge: 5
       Found 6 accounts to purge, which exceeds the configured limit of 5. Abandoning run.
+      Custom/DeletedAccountPurger/SoftDeletedAccounts: #{edap.send(:soft_deleted_accounts).count}
+      Custom/DeletedAccountPurger/AccountsPurged: 0
+      Custom/DeletedAccountPurger/AccountsQueued: 0
+      Custom/DeletedAccountPurger/ManualReviewQueueDepth: #{QueuedAccountPurge.all.count}
+      Purged 0 accounts.
+      🕐 00:00:00
```

Comment:
Note: Times are all zero in tests because I've frozen time for this whole test class.